modules/stanford_profile_styles/lib/scss/node-types/stanford_page.scss

```diff
@@ -32,6 +32,18 @@
         @include padding(null null null 80px);
       }
     }
+
+    // Main region needs to fill the whole parent in case there is not enough content to push
+    // it out.
+    .main-region {
```

Comment:
🤔 This may belong in jumpstart_ui if we are fixing a bug on the template containers. 

https://github.com/SU-SWS/jumpstart_ui/blob/8.x-1.x/templates/layouts/three-column.html.twig#L1-L6
This block of logic may need to change to provide a `flex-lg-12-of-12` when there is no left/right content.

Reply:
@jenbreese I think this PR should take care of it: https://github.com/SU-SWS/jumpstart_ui/pull/43

Reply:
Hi @sherakama It did resolve it.  This pull request can be closed as a not do.